packages/Tag/src/Tag.styles.js

```diff
@@ -0,0 +1,158 @@
+import styled, { css } from "styled-components";
+import tokens from "@paprika/tokens";
+import RawButton from "@paprika/raw-button";
+import { fontSize } from "@paprika/stylers/lib/helpers";
+import { truncateText } from "@paprika/stylers/lib/includes";
+
+const tagColorStyles = {
+  black: css`
+    background: ${tokens.color.black};
+  `,
+
+  blue: css`
+    background: ${tokens.color.blue};
+  `,
+
+  grey: css`
+    background: ${tokens.color.blackLighten70};
+    color: ${tokens.color.black};
+  `,
+
+  green: css`
+    background: ${tokens.color.greenDarken10};
+  `,
+
+  orange: css`
+    background: ${tokens.color.orangeDarken10};
+  `,
+
+  noRisk: css`
+    background: ${tokens.color.blackLighten70};
+    color: ${tokens.color.black};
+  `,
+
+  lightBlue: css`
+    background: ${tokens.color.blueLighten50};
+    color: ${tokens.color.blueDarken20};
+  `,
+
+  lightOrange: css`
+    background: ${tokens.color.orangeLighten40};
+    color: ${tokens.color.orangeDarken20};
+  `,
+
+  lowRisk: css`
+    background: #299a7a;
+  `,
+
+  mediumRisk: css`
+    background: #c9af28;
+  `,
+
+  highRisk: css`
+    background: #cd3c44;
+  `,
+
+  alert: css`
+    background: none;
+    color: ${tokens.color.orangeDarken10};
+  `,
+};
+
+export const Tags = styled.ul(() => {
+  return css`
+    display: flex;
+    flex-wrap: wrap;
+    line-height: 1;
+    margin: 0;
+    padding: 0;
+  `;
+});
+
+const borderColorStyles = ({ borderColor }) => css`
+  border: 1px solid ${borderColor};
+`;
+
+const tagStyles = ({ tagColor, borderColor }) => css`
+  align-items: center;
+  background: ${tokens.color.blackLighten70};
+  ${borderColor ? borderColorStyles : ""}
+  border-radius: ${tokens.space[0] * 2}px;
+  color: ${tokens.color.white};
+  display: flex;
+  line-height: 1;
+  margin-bottom: 2px;
+  margin-right: ${tokens.spaceSm};
+  max-width: 100%;
```

Comment:
I think a `maxWidth` prop might be a better way to handle truncation.  The default value could still be `100%` though.

Reply:
Ok, as we discussed we can leave this to the consumer to dictate as there may be other surrounding elements such as `<li>` that also need a width set on them. We can revisit in a version as well when we could add a surrounding context in the `tags` component (which is exported with `Tag`